### DISCUSSION TITLE
add extra icons

### DISCUSSION THREAD

**Discussion by aghili371** [OrchardCMS/OrchardCore#15889] [Q&A]
it's possible to add new icons?

i need to add medical icons. this package:
[https://github.com/samcome/webfont-medical-icons](https://github.com/samcome/webfont-medical-icons)

**Comment by sebastienros** [comment]
Where?

**Comment by aghili371** [comment]
> Where?

in icon picker field.

**Comment by sebastienros** [comment]
You could add a menu item by code, which would let you specify exactly what you need, but right now there is no way to have other ones than the fontawesome ones.

**Comment by sebastienros** [comment]
I am not closing it because it would make sense to make it extensible. If anyone is interested in implementing we'll take some time to define some implementation recommendations.

**Comment by aghili371** [comment]
that will be great if we can upload the font file or write down the URL of the file  and save the css class like " fa fa-back" of "flaticon-human-liver "  in the field value.
i had a look on "OrchardCore.ContentFields" , you are using the JSON array for list of icons. so i think it possible to inject new items in this list.

`   t.defaultOptions = c.extend(t.defaultOptions, {
        icons: [ {
            title: "fab fa-500px",
            searchTerms: []
        }, {
            title: "fab fa-accessible-icon",
            searchTerms: [ "accessibility", "handicap", "person", "wheelchair", "wheelchair-alt" ]
        },`
